packages/tupaia-web/src/features/Dashboard/ExportDashboard/Preview.tsx

```diff
@@ -0,0 +1,248 @@
+import { Pagination } from '@material-ui/lab';
+import { Button, LoadingContainer, Checkbox as BaseCheckbox } from '@tupaia/ui-components';
+import { useEntity, useProject } from '../../../api/queries';
+import { useExportDashboard } from '../../../api/mutations';
+import { PDFExport } from '../../../views';
+import { MOBILE_BREAKPOINT } from '../../../constants';
+
+const ButtonGroup = styled.div`
+  padding-top: 2.5rem;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const PrimaryContext = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  width 100%;
+  align-items: start;
+`;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  flex-basis: 83.3333%;
+
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    flex-direction: row;
+  }
+`;
+
+const ExportSettingsContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    width: 62%;
+    margin-right: 2rem;
+    margin-bottom: 0;
+  }
+`;
+
+const ExportSetting = styled.div`
+  border: ${({ theme }) => theme.palette.text.secondary};
+  border-width: 0.1rem;
+  border-style: solid;
+  border-radius: 10px;
+
+  .MuiFormGroup-root {
+    align-content: start;
+  }
+
+  .MuiFormControlLabel-label {
+    font-size: 0.875rem;
+  }
+
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  padding: 1rem;
+`;
+
+const Legend = styled.legend`
+  color: ${({ theme }) => theme.palette.text.primary};
+  font-size: 1rem;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+const Checkbox = styled(BaseCheckbox)`
+  margin: 0.5rem 0 0 1rem;
+  .MuiButtonBase-root {
+    padding: 0;
+    margin-right: 0.5rem;
+  }
+  label {
+    padding: 0.5rem 0 0 0.5rem;
+  }
+`;
+
+const PreviewPanelContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    width: 36%;
+  }
+`;
+
+const ExportSettingsInstructionsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  padding-bottom: 0.7rem;
+`;
+
+const PreviewHeaderContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 0.3rem;
+`;
+
+const PreviewPagination = styled(Pagination)`
+  .MuiPaginationItem-page {
+    font-size: 0.7rem;
+  }
+`;
+
+const PreviewContainer = styled.div`
+  display: flex;
+  background-color: white;
+  height: 30rem;
+  min-width: 20rem;
+  overflow: auto;
+`;
+
+const Instructions = styled(Typography)`
+  color: ${({ theme }) => theme.palette.text.primary};
+  font-size: 0.875rem;
+  line-height: 1.4;
+`;
+
+const PreviewTitle = styled(Typography)`
```

Comment:
```suggestion
const PreviewTitle = styled(Typography).attrs({
    variant: "h2"
})`
```

Since this is acting as a title, it should be an `h2` or potentially an `h3` (depending on whether there is an `h2` above it or not). This is mainly because by using heading levels, people who use assistive technologies can navigate through the web page by heading level

Reply:
Ah thanks, I never knew you could do this :100: